azure-pipelines.yml

```diff
@@ -135,3 +135,38 @@ jobs:
         tasks: 'assembleDebug'
     displayName: 'Build androidJNI'
     
+- job: androidNativeActivity
+  pool:
+    vmImage: 'macOS-10.14'
+    
+  steps:
+  - script: |
+      git submodule update --init --recursive
+    displayName: 'Checkout dependencies'
+  - script: |
+      set -eu
+      curl -Ls https://github.com/ninja-build/ninja/releases/download/v1.9.0/ninja-mac.zip -o ninja-mac.zip
+      unzip ninja-mac.zip
+      sudo cp -v ninja /usr/local/bin/
+    displayName: 'Install Ninja'
+  - script: |
+      npm install --prefix . v8-android@7.5.1
```

Comment:
Shouldn't this go in the cmake for napi?

Reply:
yes, I think it's better for the users to let cmake do that work. But I don't think I'll do it in this PR tho.

Reply:
I'll do it in this PR actually.